manager/integration/tests/test_cloning.py

```diff
@@ -36,49 +94,178 @@ def test_cloning_without_backing_image():  # NOQA
             - ReadWriteOnce
           resources:
             requests:
-              storage: 10Gi
+              storage: 3Gi
         ```
-    5. Specify the `cloned-pvc` in a cloned pod yaml and deploy the cloned pod
-    6. Wait for the `CloneStatus.State` in `cloned-pvc` to be `completed`
-    7. In 3-min retry loop, wait for the cloned pod to be running
-    8. Verify the data in `cloned-pvc` is the same as in `source-pvc`
-    9. In 2-min retry loop, verify the volume of the `clone-pvc` eventually
+    5. Wait for the `CloneStatus.State` in `cloned-pvc` to be `completed`
+    6. Clone volume should get detached after cloning completion, wait for it.
+    7. Specify the `cloned-pvc` in a cloned pod yaml and deploy the cloned pod
+    8. In 3-min retry loop, wait for the cloned pod to be running
+    9. Verify the data in `cloned-pvc` is the same as in `source-pvc`
+    10. In 2-min retry loop, verify the volume of the `clone-pvc` eventually
```

Comment:
Just want to double check, are we cleaning up the pods and the PVC at the end of the test?

Reply:
Yes, the pods and pvc are getting cleaned up in the fixtures.